src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -109,6 +118,9 @@ public String runKubectlCommand(String command, ImmutableList<String> args)
               .add("--kubeconfig")
               .add(kubeConfigFile.getRemote())
               .add(command);
+      if (!namespace.equals("")) {
```

Comment:
Prefer !Strings.isNullOrEmpty()

Reply:
It's not possible for it to be null here because of the precondition in the builder but would `!namespace.isEmpty()` be a fine substitute here?

Reply:
SGTM